### PR TITLE
link fixing #479

### PR DESCRIPTION
**Pull request by ShadmanIshraq**
link fixing #479

### CODE REVIEW

File: inst/pages/04_containers.qmd [hunk 1]
```diff
@@ -736,7 +736,7 @@ GlobalPatterns_phyloseq2
 
 Conversion is possible between other data formats. Interested readers can refer to the following functions:
 
-* [makeTreeSummarizedExperimentFromDADA2](https://microbiome.github.io/mia/reference/makeTreeSummarizedExperimentFromDADA2.html)  
-* [makeSummarizedExperimentFromBiom](https://microbiome.github.io/mia/reference/makeSummarizedExperimentFromBiom.html)  
+* [makeTreeSummarizedExperimentFromDADA2](https://microbiome.github.io/mia/reference/makeTreeSEFromDADA2.html)  
+* [makeSummarizedExperimentFromBiom](https://microbiome.github.io/mia/reference/makeTreeSEFromBiom.html)
```
Comment (antagomir) [inline inst/pages/04_containers.qmd]: Is this correct? The link should point to a function that has the same name than the link. Here the link name and URLs are different?

Comment (ShadmanIshraq) [reply inst/pages/04_containers.qmd]: I have changed the link names according to the URLs.

**Comment by TuomasBorman** [comment]
The function names are going to change

File: inst/pages/04_containers.qmd (hunk 1 above)
Comment (antagomir) [reply inst/pages/04_containers.qmd]: Here the link names and URLs are still not identical 

makeTreeSummarizedExperimentFromDADA2
makeTreeSEFromDADA2.html

makeSummarizedExperimentFromBiom
makeTreeSEFromBiom.html

Comment (ShadmanIshraq) [reply inst/pages/04_containers.qmd]: I have changed it. Please refer to the new commit.

**Review by antagomir — APPROVED** [review]

**Pull request merged**